### ISSUE TITLE
COLLAB 

### ISSUE THREAD

**Issue by deepearthgo** [weihua916/powerful-gnns#7]
Dear Author, 

I could reproduce task including MYTAG,PTC,NCl1,PROTEINS,IMDB-B,IMDB-M,RDT-M%K using the parameters of your paper, however, I could not reproduce the results for RDT-B and COLLAB using the parameters.

Might I get your advice or suggestion about reproducing the results of COLLAB and RDT-B.

Regards

**Comment by LYF14020510036** [comment]
hello, the code does not have a validation set, how to get the test set result.

**Comment by deepearthgo** [comment]
I reviewer the closed topic and try the --degree_as_tag

**Comment by mulinmeng** [comment]
> Dear Author,
> 
> I could reproduce task including MYTAG,PTC,NCl1,PROTEINS,IMDB-B,IMDB-M,RDT-M5K using the parameters of your paper, however, I could not reproduce the results for RDT-B and COLLAB using the parameters.
> 
> Might I get your advice or suggestion about reproducing the results of COLLAB and RDT-B.
> 
> Regards

Hi, I want to ask how do you reproduce the results? Like the MYTAG, I tried all the 
combinations of the paper's mentioned hyper-parameters......

Thx!!!!!!!!!!!!!

**Comment by weihua916** [comment]
Two quick points you can confirm are:

- Have you correctly specified --graph_pooling_type? We used the sum graph pooling for bio/chemistry datasets and the mean graph pooling for social network datasets.
- Have you added --deg_as_tag (use the node degree as the input node feature) for IMDB and COLLAB?
